AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -208,6 +208,7 @@ gunosy.com##.download_link
 gunosy.com##.airplain
 gunosy.com##.store_fixed_footer
 dospara.co.jp##.p-footer-app
+kurashiru.com###footer_banner_app
```

Comment:
According to the code, it is an ad. Will move it.

Reply:
Fixed https://github.com/AdguardTeam/AdguardFilters/commit/7201947ba1f64c80710666d1adb5e6e17424a8a7

Reply:
ok. thank you